src/game/sound.c

```diff
@@ -51,13 +51,13 @@ static int16_t m_AmbientLookup[MAX_AMBIENT_FX] = { 0 };
 static int32_t m_AmbientLookupIdx = 0;
 
 static SOUND_SLOT *Sound_GetSlot(
-    int32_t sfx_num, uint32_t loudness, PHD_3DPOS *pos, int16_t mode);
+    int32_t sfx_num, int32_t loudness, PHD_3DPOS *pos, int16_t mode);
 static void Sound_UpdateSlotParams(SOUND_SLOT *slot);
 static void Sound_ClearSlot(SOUND_SLOT *slot);
 static void Sound_ClearSlotHandles(SOUND_SLOT *slot);
 
 static SOUND_SLOT *Sound_GetSlot(
-    int32_t sfx_num, uint32_t loudness, PHD_3DPOS *pos, int16_t mode)
+    int32_t sfx_num, int32_t loudness, PHD_3DPOS *pos, int16_t mode)
```

Comment:
This change breaks some water sound on Lost Valley, start that level and go forwards until the water. You'll hear the problem right there.

Reply:
Waaat. Great find. Went back to uint32_t, but added a cast for SOUND_NOT_AUDIBLE.